src/System.Collections.Immutable/tests/project.lock.json

```diff
@@ -2,61 +2,175 @@
   "locked": true,
   "version": -9997,
   "targets": {
-    ".NETPortable,Version=v4.5,Profile=Profile259": {
-      "xunit/2.0.0-beta5-build2785": {
+    "DNXCore,Version=v5.0": {
+      "System.Collections/4.0.10-beta-22816": {
         "dependencies": {
-          "xunit.core": "[2.0.0-beta5-build2785]",
-          "xunit.assert": "[2.0.0-beta5-build2785]"
-        }
+          "System.Runtime": "4.0.20-beta-22816"
+        },
+        "compile": [
+          "lib/contract/System.Collections.dll"
+        ],
+        "runtime": [
+          "lib/aspnetcore50/System.Collections.dll"
+        ]
       },
-      "xunit.abstractions/2.0.0-beta5-build2785": {
+      "System.Diagnostics.Contracts/4.0.0-beta-23008": {
```

Comment:
Thanks.  DNX doesn't exactly have a great option for adding a new * dependency when you are locked on a version of packages that is older than the newest in your feeds.
What I have done in the past is first add the dependency as an explicit version, eg: 4.0.0-beta-22927, generate the lock file, then modify the project.json to use a * version and modify the lock file's projectFileDependencyGroups section to agree with the project.  I bet @davidfowl would cringe at this :).

I think long term we can switch to fixed stable dependencies, and then rely on some policy in the test projects to run the tests on the latest builds.